attributes/network/cddl/9_account.cddl

```diff
@@ -0,0 +1,120 @@
+    ; included in the list of owners by default, even if the sender is listed
+    ; in this list.
+    ; The server MUST return an error if any role is not supported by any
+    ; feature included in the account.
+    ? 1 => {
+        * non-anonymous-identity => [ * tstr ],
+    },
+
+    ; Required list of features and their arguments.
+    ; The server MUST return an error if no features are selected; an account
+    ; without features doesn't do anything and is an error.
+    2 => [ 1* attribute ],
+
+    ; Any other information pertinent to this account.
+    * tstr => any,
+}
+account.create@return = {
+    ; The account identity. Use this to contact the account directly.
+    0 => identity,
+}
+; end::create[]
+
+; tag::setDescription[]
+; Sets the description of an account.
+account.setDescription@param = {
+    ; The identity of the account.
+    0 => non-anonymous-identity,
+
+    ; The new name or description.
+    1 => tstr,
+}
+
+account.account.setDescription@return = ()
+; end::setDescription[]
+
+; tag::listRoles[]
+; List all the roles available for an account, based on its features.
+account.listRoles@param = ()
+
+account.listRoles@return = {
+    ; The list of roles available for identities.
+    0 => [ * account-role ],
+}
+; end::listRoles[]
+
+; tag::getRoles[]
+; List the roles of an identity.
+account.getRoles@param = {
+    ; The identity to get the roles from.
+    0 => non-anonymous-identity / [ * non-anonymous-identity ],
+}
+
+account.getRoles@return = {
+    ; The list of roles, per identity.
+    0 => {
+        identity => [ * account-role ],
+    }
+}
+; end::getRoles[]
+
+; tag::addRole[]
+; Add roles to an identity related to the account. Adding a role that is already
+; set to an identity does nothing. The only role that must be defined all the
+; time is the "owner". Other roles can be added by features and can be listed by
+; calling the `account.listRoles` endpoint.
+account.addRole@param = {
+    ; The identity to add the role to.
+    0 => non-anonymous-identity,
+
+    ; One or more roles to add.
+    1 => [ * account-role ] / account-role,
+}
+account.addRole@return = ()
+; end::addRole[]
+
+; tag::removeRole[]
+; Remove roles from an identity. If the role isn't set, this will do nothing.
+account.removeRole@param = {
+    ; The identity to remove the role to.
+    0 => non-anonymous-identity,
+
+    ; One or more roles to remove.
+    1 => [ * account-role ] / account-role,
+}
+account.removeRole@return = ()
+; end::removeRole[]
+
+; tag::info[]
+; Get the information of an account. Some information might be missing if the
+; server desires it. For example, the server might restrict which identities
+; can have the list of owners.
+account.info@param = {
+    ; The identity of the account to request the information of.
+    0 => non-anonymous-identity,
+}
+account.info@return = {
+    ; The name or description of the wallet.
+    ? 0 => tstr,
+
+    ; The list of features and their argument.
```

Comment:
We use multiple terms to relate to the same thing. We need to make this uniform and always use the same vocabulary.

Reply:
Changed the type name.